airflow/ui/src/pages/Dashboard/Stats/DAGImportErrors.tsx

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Badge, Text, Button, useDisclosure } from "@chakra-ui/react";
+import { FiChevronRight } from "react-icons/fi";
+
+import { useImportErrorServiceGetImportErrors } from "openapi/queries";
+
+import { DAGImportErrorsModal } from "./DAGImportErrorsModal";
+
+export const DAGImportErrors = () => {
+  const { onClose, onOpen, open } = useDisclosure();
+
+  const { data, error } = useImportErrorServiceGetImportErrors();
+  const importErrorsCount = data?.total_entries ?? 0;
+  const importErrors = data?.import_errors ?? [];
+
+  return (
+    importErrorsCount > 0 && (
+      <Box alignItems="center" display="flex" gap={2}>
+        <Button
+          alignItems="center"
+          borderRadius="md"
+          display="flex"
+          gap={2}
+          onClick={onOpen}
+          variant="outline"
+        >
+          <Badge background="red" borderRadius="full" color="white" px={2}>
```

Comment:
```suggestion
          <Badge background="red.solid" borderRadius="full" color="white" px={2}>
```

Let's make sure to use the correct semantic tokens.

Reply:
Actually we can just do `colorPalette="red"` and let the badge component handle the rest

Reply:
Yaa, we can do, But in that case the bg will light red and number will displayed in red.
It would be different from screenshot attached to issue.

Reply:
Ok nevermind then.